docs/articles/documentation/reference/test-api/global/clientfunction.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+title: ClientFunction Constructor
+permalink: /documentation/reference/test-api/global/clientfunction.html
+---
+# Client Function Constructor
+
+You can use `ClientFunction` constructor to create a client function.
+
+```text
+ClientFunction( fn [, options] )
+```
+
+Parameter              | Type     | Description
+---------------------- | -------- | ---------------------------------------------
+`fn`                   | Function | A function to be executed on the client side.
+`options`&#160;*(optional)* | Object   | See [Options](#options).
+
+> Client functions cannot return DOM nodes. Use [selectors](../selecting-page-elements/selectors/README.md) to achieve this instead.
```

Comment:
Change the second sentence to:
Use [selectors](../selecting-page-elements/selectors/README.md) instead.